### ISSUE TITLE
External Display not working on Legion Go in Game Mode as of 3.5

### ISSUE THREAD

**Issue by VorpalQ** [ublue-os/bazzite#1309]
### Describe the bug

When I power on my Go connected to my dock, my external display functions normally, but once game mode starts, my internal display turns off as normal, but the external goes off as well. This issue is *not* present in Desktop mode, external display(s) function normally there. It is game mode only. If I am in Desktop Mode and switch to Game Mode, the same issue occurs, so it is not just on boot.

### What did you expect to happen?

Prior to this update. my external display would function as the sole display when my Legion Go was docked.

### Output of `rpm-ostree status`

```shell
State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:stable
                   Digest: sha256:dcf0c28f284a613759ea0b1da0c10ba8744dc13f2de90fa5de5d6be49f0a01f5
                  Version: 40.20240630.0 (2024-06-30T20:42:59Z)
          LayeredPackages: mullvad-vpn

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:stable
                   Digest: sha256:b48da348e0f942a18e833fb14f5483d70b291b41f34089c390148ad53a36dec4
                  Version: 40.20240625.0 (2024-06-27T02:01:04Z)
          LayeredPackages: mullvad-vpn
```

### Hardware

Lenovo Legion Go - 83E1

### Extra information or context

Everything functioned normally until the update that I installed today (07/01/24)

**Comment by VorpalQ** [comment]
After further testing, on the 06/30 image, if I connect via USB-C to USB-C *directly* to a monitor, the display out works properly. Using any USB-C to HDMI dongle or dock, it does not. I have tried three different docks (Anker, Ugreen, Mokin) and all have this issue with the HDMI in Game Mode.

**Comment by abpages** [comment]
I have an issue as well using Xreal Air1's. They do not work in Bazzite version 40, I had to rollback to 39 in order to get them to work in Game mode.

**Comment by VorpalQ** [comment]
My issue has been resolved in the Version: 40.20240703.0 (2024-07-03T07:43:35Z) build of Bazzite. Closing this issue.
